.github/workflows/pytest.yaml

```
@@ -10,7 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: ./zammad-ai-workflow
+        working-directory: 
+        - ./zammad-ai-workflow
```

actionlint (1.7.12)

[error] expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)